### ISSUE TITLE
[Web] Can't build games bigger than 2 GB

### ISSUE THREAD

**Issue by DipeshAggarwal** [renpy/renpy#3976]
Hey PyTom.

If I try to build a web version of my game (the game is 3.2 GB), I get this error.
```
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/web.rpy", line 362, in <module>
  File "game/web.rpy", line 243, in build_web
  File "game/web.rpy", line 156, in repack_for_progressive_download
BadZipfile: Bad magic number for central directory

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "launcher/game/web.rpyc", line 362, in script
  File "renpy/ast.py", line 1111, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 1049, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/web.rpy", line 362, in <module>
  File "game/web.rpy", line 243, in build_web
  File "game/web.rpy", line 156, in repack_for_progressive_download
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python2.7/zipfile.py", line 793, in __init__
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python2.7/zipfile.py", line 862, in _RealGetContents
BadZipfile: Bad magic number for central directory

Windows-10-10.0.22000 AMD64
Ren'Py 7.5.0.22062402
Ren'Py Launcher 7.5.0.22062402
Tue Sep 13 09:20:41 2022
[/code]
```

**Comment by renpytom** [comment]
Reading this, you're creating a game that would require the player to download 3.2 GB into their browser to start playing. While I could fix the underlying issue, realize that you're doing something majorly wrong here.

**Comment by DipeshAggarwal** [comment]
Most of the 3.2 GB is marked as a progressive download. So the player will initially only download 35 MB

**Comment by renpytom** [comment]
Can you try this on the nightly? I believe this has been fixed by 966cce804e847105643371aba51151e075906928 .
